src/main/kotlin/no/skatteetaten/aurora/gobo/integration/boober/EnvironmentService.kt

```diff
@@ -0,0 +1,58 @@
+package no.skatteetaten.aurora.gobo.integration.boober
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import mu.KotlinLogging
+import no.skatteetaten.aurora.gobo.graphql.applicationdeployment.ApplicationDeploymentRef
+import org.springframework.stereotype.Service
+
```

Comment:
Her er det nok litt generel behov for refactoring for readability. Se på navngivning av data klassene, men kanskje også skille noe ut i noen metoder som beskriver litt mer hva som skjer.

Reply:
enig i det